packages/cli/src/commands/seed.ts

```diff
@@ -0,0 +1,504 @@
+  if (!coreAddress) {
+    throw new Error(
+      `Unable to select a deployed BoostCore with chain ID ${chainId}`,
+    );
+  }
+
+  const core = new BoostCore({
+    config,
+    account,
+    address: coreAddress,
+  });
+
+  if (!positionals.length) throw new Error('No seed provided');
+  const templates = await Promise.all(positionals.map(getSeed));
+
+  let sharedBudget: ManagedBudget | undefined;
+  let sharedBudgetConfig: ManagedBudgetPayload | undefined;
+  const boostIds: string[] = [];
+  for (const template of templates) {
+    let budget: ManagedBudget;
+    if (typeof template.budget === 'string' && isAddress(template.budget))
+      budget = core.ManagedBudget(template.budget);
+    // TODO: create budget from Core
+    else if (sharedBudgetConfig === template.budget && sharedBudget) {
+      budget = sharedBudget;
+    } else {
+      const payload = {
+        ...template.budget,
+        authorized: [...template.budget.authorized, coreAddress],
+        roles: [...template.budget.roles, ManagedBudgetRoles.MANAGER],
+      } satisfies ManagedBudgetPayload;
+      budget = await registry.initialize(
+        crypto.randomUUID(),
+        core.ManagedBudget(payload),
+      );
+      sharedBudget = budget;
+      sharedBudgetConfig = template.budget;
+    }
+
+    const incentivePromises = template.incentives.map(async (incentive) => {
+      let amount = 0n;
+      switch (incentive.type) {
+        case 'AllowListIncentive':
+          return core.AllowListIncentive(incentive);
+        case 'ERC20Incentive':
+          if (incentive.strategy === StrategyType.RAFFLE) {
+            amount += incentive.reward;
+          }
+          if (incentive.strategy === StrategyType.POOL) {
+            amount += incentive.reward * incentive.limit;
+          }
+          if (incentive.mintableMockAsset)
+            await fundBudgetForIncentive(budget, amount, incentive.asset);
+          return core.ERC20Incentive(incentive);
+        case 'ERC20VariableCriteriaIncentive':
+          amount += incentive.limit;
+          if (incentive.mintableMockAsset)
+            await fundBudgetForIncentive(budget, amount, incentive.asset);
+          return core.ERC20VariableCriteriaIncentive(incentive);
+        case 'ERC20VariableIncentive':
+          amount += incentive.limit;
+          if (incentive.mintableMockAsset)
+            await fundBudgetForIncentive(budget, amount, incentive.asset);
+          return core.ERC20VariableIncentive(incentive);
+      }
+    });
+
+    const incentives = await Promise.all(incentivePromises);
+
+    const boost = await core.createBoost({
+      protocolFee: template.protocolFee,
+      maxParticipants: template.maxParticipants,
+      budget: budget,
+      action: core.EventAction(template.action as EventActionPayload),
+      validator: core.SignerValidator(template.validator),
+      allowList: await getAllowList(template, { core }),
+      incentives,
+    });
+    boostIds.push(boost.id.toString());
```

Comment:
boost ids are probably fine for now, but this makes me want to write a `boost.serialize` that returns a boost to a shape like boostconfig

Reply:
I think a simple Boost Id is most immediately helpful for frontend devs looking for state but yes I agree.